frontend/src/routes/+page.svelte

```diff
@@ -97,11 +106,8 @@ import { onMount } from 'svelte';
 </section>
 
 <aside class="has-background-primary-light">
-    <div class="level is-mobile text--small--uppercase py-4">
-        {#each Array(16) as _}
-        <div class="level-item">Registration is open</div>
-        <div class="level-item">🔥</div>
-        {/each}
+    <div id='marquee' class="marquee level is-mobile text--small--uppercase py-4">
+        <div>Registration is open&nbsp;🔥&nbsp;</div>
```

Comment:
This text is provided by the user on the CMS, so we can't just rely on them using non-breaking spaces. A margin-left will serve, maybe? If it's absolutely necessary, we can just replace all spaces of the string with non-breaking ones and append an extra space at the end:

```js
marqueeText.trim().replace(/\s/g, '\xa0') + '\xa0'
```

Reply:
Fixed! Now, I use a `1ch` padding right for each element. This makes one character space between each element. I think that there is not one _technical solution_ for that, is a design solution.